packages/CollapsibleCard/src/components/Header/Header.styles.js

```diff
@@ -43,6 +43,8 @@ export const Header = styled.div(
       }
     }
     &:focus-visible {
+      ${stylers.focusRing(true)}
+      ${stylers.zValue(1)}
       box-shadow: ${tokens.highlight.active.withBorder.boxShadow};
```

Comment:
I think the `stylers.focusRing(true)` makes this `box-shadow` rule redundant?

ps I appreciate the use of the `stylers` helper 💖